CHANGELOG.md

```diff
@@ -1,3 +1,87 @@
+### citus v11.2.0 (January 30, 2023) ###
+
+* Adds support for outer joins having a recurring rel in the outer side of the join (e.g., \<reference table\> LEFT JOIN \<distributed table\>)
+
+* Adds support for creating `PRIMARY KEY`s and `UNIQUE`/`EXCLUSION`/`CHECK`/`FOREIGN KEY` constraints via `ALTER TABLE` command without providing a constraint name
+
+* Adds support for using identity columns on Citus managed tables
+
+* Adds `citus_task_wait()` UDF to wait on desired task status
+
+* Adds `source_lsn`, `target_lsn` and `status` fields into `get_rebalance_progress()`
+
+* Adds `citus_get_transaction_clock()` UDF that returns a monotonically increasing logical causal clock for the current transaction
+
+* Allows `citus_update_node()` to work with nodes from different clusters
+
+* Introduces `citus.propagate_session_settings_for_loopback_connection` GUC to propagate local settings to new connections
```

Comment:
we decided to lower the visibility of this (e.g., it is GUC_NO_SHOW_ALL). We do that because there are several edge cases with this GUC, and it is mostly useful for debugging / testing purposes for now.

So, maybe move this to below, and reword it to tell this is for testing/debugging purposes in the desc..

Reply:
hmm, then maybe we shouldn't mention this at ll ?